compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -99,6 +99,22 @@ impl Attribute {
         }
     }
 
+    pub fn is_path(&self, name: &[Symbol]) -> bool {
+        match &self.kind {
+            AttrKind::Normal(normal) => {
+                normal.item.path.segments.len() == name.len()
+                    && normal
+                        .item
+                        .path
+                        .segments
```

Comment:
This ignores the [args](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_ast/ast/struct.PathSegment.html#structfield.args) of each path segment -- should we be checking that those are empty?